### PR TITLE
Doronst/bio 548/track more files with lfs

### PR DESCRIPTION
**Pull request by doron-st**

### CODE REVIEW

**Comment by doron-st** [comment]
It's kind of impossible to test that the size of the clone was reduced.
When I clone into a machine without git-lfs, it clones the master branch (170M)
When I checkout this branch it is still 170M since I guess it doesn't delete the files which were already downloaded.

Anyways, git-lfs will not delete the files which were already uploaded to remote.
We need more involved migration if we want to do that. And I rather we avoid it.

Still, I hope that after we merge this, and re-clone the size of the clone folder will be reduced.

File: .gitattributes [hunk 1]
```diff
@@ -5,12 +5,17 @@
...
+*.hdf filter=lfs diff=lfs merge=lfs -text
+*.parquet filter=lfs diff=lfs merge=lfs -text
 *.pkl filter=lfs diff=lfs merge=lfs -text
 *.bw filter=lfs diff=lfs merge=lfs -text
 *.interval_list filter=lfs diff=lfs merge=lfs -text
 *.bedGraph filter=lfs diff=lfs merge=lfs -text
 *.fasta filter=lfs diff=lfs merge=lfs -text
 *.h5 filter=lfs diff=lfs merge=lfs -text
+*.sdf/* filter=lfs diff=lfs merge=lfs -text
+*.json filter=lfs diff=lfs merge=lfs -text
```
Comment (ilyasoifer) [inline .gitattributes:19]: Does not make sense to add json or txt to lfs, no?

File: .gitattributes [hunk 2]
```diff
@@ -5,12 +5,17 @@
 *.bai filter=lfs diff=lfs merge=lfs -text
 *.pickle filter=lfs diff=lfs merge=lfs -text
 *.tbi filter=lfs diff=lfs merge=lfs -text
-*tsv filter=lfs diff=lfs merge=lfs -text
-*parquet filter=lfs diff=lfs merge=lfs -text
-*bed filter=lfs diff=lfs merge=lfs -text
+*.csv filter=lfs diff=lfs merge=lfs -text
+*.tsv filter=lfs diff=lfs merge=lfs -text
```
Comment (ilyasoifer) [inline .gitattributes:9]: Do we actually have large csv and tsv files?  Even sdf that we have is not really big

**Review by ilyasoifer — COMMENTED** [review]
I am a bit surprised that this was necessary.

**Pull request closed without merging**